noir-projects/aztec-nr/slow-updates-tree/src/slow_map.nr

```diff
@@ -17,6 +17,7 @@ fn compute_next_change(time: Field) -> Field {
     ((time as u120 / EPOCH_LENGTH + 1) * EPOCH_LENGTH) as Field
 }
 
+// TODO(#4760): Rename slow updates to shared mutable and ideally move the impl to state-vars in aztec-nr.
```

Comment:
As part of #4761 we can likely nuke this entirely. ![Added via Giphy](https://media0.giphy.com/media/rhYsUMhhd6yA0/giphy.gif?cid=c9487743uyalnfm1agakea65qkuncuuksvmbg7v665t4tdl7&ep=v1_gifs_search&rid=giphy.gif&ct=g)